object.go

```diff
@@ -478,7 +478,8 @@ func (d *Decoder) getStructDefByIndex(idx int) (reflect.Type, classInfo, error)
 	cls = d.classInfoList[idx]
 	s, ok = getStructInfo(cls.javaName)
 	if !ok {
-		return nil, cls, perrors.Errorf("can not find go type name %s in registry", cls.javaName)
+		return nil, cls, nil
```

Comment:
the default behavior will change to skip POJO not registered, while it will return error before. There is not a switch.

Reply:
oh， I will add a switch.